apps/prairielearn/src/tests/fileEditor.test.ts

```diff
@@ -784,17 +751,8 @@ function doEdits(data) {
     // (A, A, A*, A*)
 
     editPost('save_and_sync', data.contentsC, data.url, true, false, null);
-    // (A, A, A*, A*)
-
-    waitForJobSequence(locals, 'Error');
-    pullInLive();
-    // (A, A, A, A)
-
-    editGet(data.url, false, false, data.contentsA, null);
-    // (A, A, A, A)
-
-    editPost('save_and_sync', data.contentsC, data.url, true, false, null);
-    // (C, C, A, C)
+    waitForJobSequence(locals, 'Success');
+    // (C, C, A*, C)
```

Comment:
Change in behavior! `save_and_sync` will now pull changes, so this will no longer error.  This makes some of the following deleted steps unnecessary, as there's no failure to recover from.

Reply:
Makes sense.